### PR TITLE
Revised documentation

### PR DESCRIPTION
**Pull request by HungWoeiNeoh**

### CODE REVIEW

File: savvy_scheduler.md [hunk 1]
```diff
@@ -57,15 +54,13 @@ The scheduling algorithms you are going to implement are:
...
-
-**You should use the priority queue that we provided to help you complete this part of the lab.**
+Make sure you understand these schemes before implementing them. Please refer to the lectures or [the scheduling chapter](http://cs241.cs.illinois.edu/coursebook/Scheduling) in the coursebook for details.
 
-To complete this lab, you must implement the six comparator functions and eight scheduler functions (and one optional one) defined in `libscheduler.c`. These functions are self-descriptive, but a full function outline for each function is provided for you in the file. These functions will be utilized by the green threading library (Check out the section on Gthreads below).
+## Directions
 
-You might want to understand how scheduler works. So we put a detailed explanation in the bottom of this webpage.
+To complete this lab, you must implement comparator functions and scheduler functions defined in `libscheduler.c`. These functions are self-descriptive, but a full function outline for each function is provided for you in the header file. These functions will be utilized by the green threading library (Check out the section on Gthreads below).
```
Comment (rafilong) [inline savvy_scheduler.md]: Nit: `comparator and scheduler functions` ? Current version works too

File: savvy_scheduler.md [hunk 2]
```diff
@@ -82,98 +77,33 @@ The second part of the lab is to set up scheduler itself and manage incoming job
...
-
-### int comparer_rr(const void \*a, const void \*b) 
-
-Comparer function for rr scheme
-
-### int comparer_sjf(const void \*a, const void \*b) 
-
-Comparer function for sjf scheme
-
-### void scheduler_new_job(job \*newjob, int job_number, double time, scheduler_info \*sched_data)
```
Comment (rafilong) [inline savvy_scheduler.md]: IMO understanding the use of the new_job, quantum_expired, and job_finished functions was the most confusing part of this lab. Unless I'm missing where you added this back elsewhere, I feel like having this documentation is useful here.

Comment (HungWoeiNeoh) [reply savvy_scheduler.md]: I plan to add this into the `.h` files. Comments like these should all be in the header files, and not split up between the docs and the `.h` files

Comment (rafilong) [reply savvy_scheduler.md]: Good point, definitely agree.

**Pull request merged**